tgui/packages/tgui/styles/colors.scss

```diff
@@ -29,6 +29,8 @@ $average: #f08f11 !default;
 $bad: #db2828 !default;
 $label: #7e90a7 !default;
 
+$gold: #f2a918 !default;
```

Comment:
Probably move it closer to base colors, after yellow?

Reply:
Maybe. but the logic is:
Base colours => State colours => Custom colours